packages/core/strapi/lib/utils/get-dirs.js

```diff
@@ -11,7 +11,13 @@ const getDirs = root => ({
   policies: join(root, 'src', 'policies'),
   middlewares: join(root, 'src', 'middlewares'),
   config: join(root, 'config'),
-  public: join(root, 'public'),
+  get public() {
```

Comment:
This API being internal-only at the moment, I don't think it is necessary to go through a deprecation.